std/testing/bench.ts

```diff
@@ -166,13 +197,22 @@ export async function runBenchmarks({
   if (failed) {
     setTimeout((): void => exit(1), 0);
   }
+
+  const benchmarkRunResult = {
+    success: !failed,
+    measured,
+    filtered,
+    results: benchmarkResults,
+  };
+
+  return benchmarkRunResult;
 }
 
 /** Runs specified benchmarks if the enclosing script is main. */
 export function runIfMain(
```

Comment:
Can you get rid of `runIfMain()`? It was a silly abstraction which we has in lots of other places and then removed, seems this one was missed.

Reply:
Sure.